includes/login.php

```diff
@@ -215,7 +215,7 @@ function pmpro_lostpassword_url( $redirect = '' ) {
 		return wp_lostpassword_url( $redirect );
 	}
 
-	$args = array( 'action' => 'lostpassword' );
+	$args = array( 'action' => 'reset_pass' );
```

Comment:
Also made this change, we don't use 'lostpassword' at all, so set it to use our param of 'reset_pass'